vite.config.ts

```diff
@@ -0,0 +1,22 @@
+import { resolve } from 'path';
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  build: {
+    rollupOptions: {
+      input: {
+        main: resolve(__dirname, 'example.html'),
+      },
+      output: {
+        entryFileNames: 'app.js',
+        manualChunks: undefined,
+        inlineDynamicImports: true,
+      },
+    },
+    target: 'es2021',
```

Comment:
@joshreisner  when I add here:
`copyPublicDir: false,`
It doesn't copy logo.svg and a few pointless files in `/dist` folder. Was just poking at various config options.

Reply:
thanks - `logo.svg` is needed though for the https://tsml-ui.code4recovery.org/ homepage 👍

Reply:
Oh! This is the public page, neat, makes sense.